scripts/openroad/rcx.tcl

```diff
@@ -11,28 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+source $::env(SCRIPTS_DIR)/openroad/common/io.tcl
+read
```

Comment:
looks like it was reading `$::env(RCX_LIB)` before

Reply:
it might be the case that there is a different odb using `RCX_LIB` and `RCX_LEF` but I didn't get to that yet.